src/CommandingOfficers/Tech.java

```diff
@@ -197,7 +197,7 @@ protected void revert(MapMaster gameMap)
         final int maxPercent = UnitModel.MAXIMUM_HP*10;
         for( Unit u : unitsOverCharged.get(myCommander) )
         {
-          final int preciseHP = (int) (u.getPreciseHP() * 10);
+          final int preciseHP = u.health;
```

Comment:
Maybe I should stop complaining about places where `health` and  `HP` are interchanged, despite being off from each other by an order of magnitude. Maybe.

Reply:
They're not anymore. That's the point.

Reply:
Okay. If they are the same now (which is good!) then the random magic factors of ten I see littered around are perhaps more confusing.

Reply:
The issue is that a lot of our code really wants numbers in [0,10], so now I have to divide by 10. It feels weird to define a constant for "ten percent" or something.
I dunno, I feel like it's clear enough when HP/health is known to be nominally in [0,100]